cabal-install/src/Distribution/Client/Init/Types.hs

```diff
@@ -455,7 +455,7 @@ newtype BreakException = BreakException String deriving (Eq, Show)
 instance Exception BreakException
 
 -- | Used to inform the intent of prompted messages.
-data Severity = Log | Info | Warning | Error deriving (Eq, Show)
+data Severity = Info | Warning | Error deriving (Eq, Show)
```

Comment:
Let's not derive the `Show` instance here, and do the `displaySeverity` thing we discussed on the ticket.

Reply:
Sure, let's do this